jpo-deduplicator/src/main/resources/application.yaml

```diff
@@ -34,23 +37,18 @@ kafkaTopicOdeTimJson: topic.OdeTimJson
 kafkaTopicDeduplicatedOdeTimJson: topic.DeduplicatedOdeTimJson
 enableOdeTimDeduplication: ${ENABLE_ODE_TIM_DEDUPLICATION:true}
 
-# Ode Raw Encoded Tim Json Configuration
-kafkaTopicOdeRawEncodedTimJson: topic.OdeRawEncodedTIMJson
-kafkaTopicDeduplicatedOdeRawEncodedTimJson: topic.DeduplicatedOdeRawEncodedTIMJson
-enableOdeRawEncodedTimDeduplication: ${ENABLE_ODE_RAW_ENCODED_TIM_DEDUPLICATION:true}
-
 # Ode Bsm Json Configuration
 kafkaTopicOdeBsmJson: topic.OdeBsmJson
 kafkaTopicDeduplicatedOdeBsmJson: topic.DeduplicatedOdeBsmJson
 enableOdeBsmDeduplication: ${ENABLE_ODE_BSM_DEDUPLICATION:true}
 odeBsmMaximumTimeDelta: 10000 # Milliseconds
 odeBsmMaximumPositionDelta: 1 # Meter
-odeBsmAlwaysIncludeAtSpeed: 1 # Meter / Second
+odeBsmAlwaysIncludeAtSpeed: 1 # x 0.02 Meter / Second (J2735 defined)
```

Comment:
(question) why does the comment say "1 # x 0.02 Meter / Second (J2735 defined)"
Isn't it just checking if it's over 1 m/s?

Reply:
Yeah you are right, I will modify the logic of the processor to interpret the message to do a conversion in the backend. However, this field being a 1 in the actual BSM does not mean 1 m/s. It means 1 0.2 m/s. And yes, that's weird.

Reply:
Oh okay, yeah that'd be nice to change. Thank you!

Reply:
Addressed. You can see the conversion has been added to the OdeBsmJsonProcessor on line 92.

Reply:
I believe it has been reverted